code/__DEFINES/security.dm

```diff
@@ -0,0 +1,18 @@
+
+/// Wanted statuses
+#define WANTED_ARREST "Arrest"
+#define WANTED_DISCHARGED "Discharged"
+#define WANTED_NONE "None"
+#define WANTED_PAROLE "Parole"
+#define WANTED_PRISONER "Incarcerated"
+#define WANTED_SUSPECT "Suspected"
+
+/// List of available wanted statuses
+#define WANTED_STATUSES(...) list(\
```

Comment:
why `(...)`

Reply:
Unsure. Figured it'd be of use since most calls to WANTED_STATUSES are because of WANTED_STATUS(). Turns out not required.